gamestonk_terminal/etf/etf_controller.py

```diff
@@ -64,7 +66,6 @@ def print_help(self):
         print("   q             quit this menu, and shows back to main menu")
```

Comment:
Instead of like
```
print
print
print
```

can we have a big str defined and then just a print at the end?